### DISCUSSION TITLE
Aleph Indexing Mechanism

### DISCUSSION THREAD

**Discussion by ByUnal** [alephdata/aleph#1483] [Q&A]
While indexing any file with Aleph, the process always takes around 13 minutes, no matter the file size is. Also, after indexing, I can't reach any information in that file by using search bar in Aleph. 
Am I doing something wrong ? What kind of indexing is that ? Thanks.

**Comment by pudo** [comment]
that's definitely not normal, the only thing that happens at a 10 minute interval is computing the statistics for specific collections (i.e. the count of names, email, etc. in them). Can you provide more details (server logs?)... as is, not sure what to advise.

**Comment by ByUnal** [comment]
Now I'm going to crawl a new file with aleph. I mean, I'm going to do another test. How can I reach the logs file that you wanted ?
Also, after `make test `,
`make ingest-test `doesn't work
![image](https://user-images.githubusercontent.com/43930582/100441214-2aca9500-30b7-11eb-8399-b86ae654b6dc.png)

**Comment by ByUnal** [comment]
Now I learned to reach server logs. But I need to know which log file you want. Elasticsearch ? Postgre ?

**Comment by pudo** [comment]
`worker`, `api`. 

Can I also request that you provide the information about your use case requested in [SUPPORT.md](https://github.com/alephdata/aleph/blob/main/SUPPORT.md)?
